packages/pools/src/constants/boostedPoolsSupportedChains.ts

```diff
@@ -0,0 +1,5 @@
+import { ChainId } from '@pancakeswap/chains'
+
+export const BOOSTED_POOLS_SUPPORTED_CHAIN_IDS = [ChainId.ARBITRUM_ONE] as const
```

Comment:
Can we not maintain this constant? Seems quite troublesome to check here if we're adding a booster. 

We usually have this supported chain id constant for UI control. Such as when we need to disable the feature on unsupported chain